contents/blog/growth-loops.md

```diff
@@ -0,0 +1,109 @@
+
+Products relying on sales are generally large contracts that need high levels of support, engagement, and customization. Offers are often tailored to the specific customer and require a large amount of time and effort. 
+
+For example, sales is a growth loop for enterprise SaaS companies (e.g. Salesforce, SAP, Microsoft) and big consultancies (e.g. McKinsey, Deloitte)
+
+## Choosing the right growth loop
+
+After reading these, you likely have some idea of what growth loop is right for your product and company. Two key questions to understand this further:
+
+1. Who are your ideal users?
+2. What benefits do those users get as part of your growth loop?
+
+Understanding your ideal user will help you decide where to put your effort for your growth loop. If your ideal user won’t pay for your product, it isn’t a good idea to use sales as your growth loop. You can create a growth loop that better aligns with what they want. Growth loops are about creating win-wins.
+
+A user needs a win-win to repeatedly take an action that fuels a product’s growth loop. They need an incentive. Smart companies build this incentive into the core of their product. For example, you gain followers by posting on social media, but that also keeps users engaged and attracts new users.
+
+The benefits your product and growth loop can provide are limitless. Finding new ways to provide this value is a great way.
+
+## Measuring your growth loop
+
+To understand your growth loop and the value it is providing, it is important to measure your chosen growth loop. There are several ways to do this; they usually involve measuring the input (action) in connection to the output (key metric). What these inputs and outputs depend on the product and the growth loop. 
+
+Here are some examples of how you can measure each of the growth loops.
+
+### Virality
+
+To measure virality, we want to know what drives users to share our product and the referral source of sign ups. To do this, we’ll want to measure our traffic and shareability. We want events for the loop of signing up, getting started, referring others, and repeating. More occurrences of each and better connections between them are good signs for our viral growth loop.
+
+### Performance marketing
+
+Performance marketing is about measuring marketing is leading to signups and revenue. We can do this by understanding customer cost of acquisition (CAC) and lifetime value (LTV). The margin between what it takes to acquire a user (CAC) and how much value we receive from them (LTV) drives the performance marketing growth loop (and the rest of the company).
+
+To get data about CAC and LTV, we’ll have to understand the sources of our traffic, conversion, and finally retention. This combined with our marketing spend and revenue enables us to calculate CAC and LTV (and grow our business and product).
```

Comment:
Could we add a bit on how to calculate CAC and LTV here? Just the basic formula would be enough.

Reply:
Added a note